### ISSUE TITLE
v1.3.0 uses empty installation on pypi

### ISSUE THREAD

**Issue by collinarnett** [eclipse-iofog/iofog-python-sdk#9]
## Problem Description
v1.3.0 on pypi has contains no installation files

## Steps to replicate
```
$: pip install iofog-python-sdk==1.3.0
$: cd /.local/lib/python2.7/site-packages/iofog_python_sdk
$: ls
__init__.py  __init__.pyc
$: python2
>>> import iofog_python_sdk.client
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named client
```

**Comment by sergerad** [comment]
This is being addressed in `v2.0.0` in the near future. For now you can run `pip install iofog`.

**Comment by sergerad** [comment]
Fixed packages found here https://pypi.org/project/iofog/
